SqlServerLib/SQLServer.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <iostream>
+#include "SQLConnection.h"
+#include "../CPPLogger/CPPLogger.h"
+#include "../Crypto/SymmetricCrypto.h"
+
+
+class SQLServer : public ISQLConnection
```

Comment:
If you don't inherit from SQLServer add keyword final
It should look like `class SQLServer final : public ISQLConnection`